pkgs/by-name/lu/lunacy/package.nix

```diff
@@ -0,0 +1,107 @@
+{ stdenv
+, lib
+, fetchurl
+, dpkg
+, makeWrapper
+, autoPatchelfHook
+, zlib
+, libgcc
+, fontconfig
+, libX11
+, lttng-ust
+, icu
+, libICE
+, libSM
+, libXcursor
+, openssl
+, imagemagick }:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "lunacy";
+  version = "9.3.3";
+
+  src = fetchurl {
+    url = "https://lcdn.icons8.com/setup/Lunacy_${finalAttrs.version}.deb";
+    hash = "sha256-boIbJA/UuiA3bmXbZXro3IhjFUh6JYsOEprq3n668xk=";
+  };
+
+  unpackCmd = ''
+    mkdir -p root
+    dpkg-deb -x $src root
+  '';
+
+  buildInputs = [
+    zlib
+    libgcc
+    stdenv.cc.cc
+    lttng-ust
+    fontconfig.lib
+
+    # Runtime deps
+    libICE
+    libSM
+    libX11
+    libXcursor
+  ];
+
+  nativeBuildInputs = [
+    dpkg
+    makeWrapper
+    autoPatchelfHook
+  ];
+
+  dontBuild = true;
+  dontStrip = true;
+
+  installPhase = ''
+    runHook preInstall
+
+    mkdir -p "$out/lib";
+    cp -R "opt/icons8/lunacy" "$out/lib"
+    cp -R "usr/share" "$out/share"
+
+    patchelf \
+      --add-needed libICE.so.6 \
+      --add-needed libSM.so.6 \
+      --add-needed libX11.so.6 \
+      --add-needed libXcursor.so.1 \
+      "$out/lib/lunacy/Lunacy"
```

Comment:
```suggestion
```
this can be removed and then add:
```nix
  runtimeDependencies = [
    # Runtime deps
    libICE
    libSM
    libX11
    libXcursor
  ];
```
somewhere.